docs/onboardee-checklist.md

```diff
@@ -83,18 +86,17 @@ you feel their work is interesting and ask them what they're doing work-wise for
 * You can read the main page of the document practices completely, and **sneak peak** into the linked children pages to get the high level image you'd need and be
 able to get back to them later upon need.
 
-- read and make sure you've understood your service's documentation
-  - see appropriate page in
-    https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/655463/Services
-  - see your project's README.md files
-  - test your service
-  - see that the docs adhere to
-    https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/753772/Documentation+practices
-  - fix missing parts
-- are you able to set up a development environment for what you're developing,
```

Comment:
Under the `Get your service up and running` section, it's written that they should get it up and running and it's actually a very important step:

> * Get the local development environment of the repository you'll be coding in up and running **using README.md**
>      * **This is an important step** because it would not make sense to start coding a feature if a new team member cannot effortlessly get the dev env up and
    running based on README.md instructions.

So, to my understanding, it is already included.

Reply:
ah, sorry for being unclear.
I meant that the criterion _when the dev env is up and running_ is that you can log in and use its functionality.
So my point is that the text you pointed out "Get your service up and running" should have a sentence like:
"Your service can be considered up and running if you can log in to your local environment (if it has login functionality) and use the service locally."
Or something similar.

Reply:
Ah I see, yeah you're right that I removed the specification of what exactly is "up and running".. My bad!